File: challenges/week5_swappin_aint_easy/solutions/swap/swap.py
def swap(a, b):
    return b, a

def main():
    a, b = 1, 2
    print(f"Before: a={a}, b={b}")
    a, b = swap(a, b)
    print(f"After: a={a}, b={b}")

if __name__ == "__main__":
    main()

File: challenges/week5_swappin_aint_easy/solutions/swap/swap.rs
fn swap(a: i32, b: i32) -> (i32, i32) {
    (b, a)
}

fn main() {
    let (a, b) = (5, 3);
    println!("Before: a={}, b={}", a, b);
    let (a, b) = swap(a, b);
    println!("After: a={}, b={}", a, b);
}
